cluster/manifests/skipper/rbac.yaml

```diff
@@ -9,6 +9,13 @@ kind: ClusterRole
 metadata:
   name: skipper-ingress
 rules:
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - ingresses
+  verbs:
+  - get
+  - list
```

Comment:
IMO would be nice to use either yaml or json array notation consistently

Reply:
yes I also thought the same and then I thought the part below we will drop after v1 migration...